src/components/IncidentList.vue

```diff
@@ -1,22 +1,53 @@
 <template>
   <div :class="$style.wrapper">
     <center-wrapper>
+      <div :class="$style.filter">
+        <select :class="$style['filter-select']" v-model="selectedCity">
+          <option :value="null" selected>FILTER BY LOCATION</option>
+          <option v-for="city in allCities" :key="city" :value="city">{{ city }}</option>
+        </select>
+      </div>
```

Comment:
@michielkoning if I move this to header, how do you recommend I share `selectedCity` between the filter and the incidents list? I saw that I could use props (which I'd likely default to despite mild drilling) or events (seems like spaghetti to me). But I wonder if you have opinions here.

Reply:
I would prefer to be the filter part of the incidentlist and not making it part of the header. It's easier for you and me to implement and the header doesn't contain elements of a specific page. This will be a slight change to the original design.